server/lib/configuration.js

```diff
@@ -126,24 +126,6 @@ const conf = module.exports = convict({
       'development'
     ]
   },
-  experiments: {
```

Comment:
Is the only part we are missing here is a config via ENV variables, like we were planning for Phase 1?

I am okay with merging this without it, and then make another PR to make stuff like `SMS stuff` configurable. Or we can make SMS 100% and add the ENV switching for some other new feature